testsuite/domain/pom.xml

```diff
@@ -178,7 +177,7 @@
                     <argLine>${jvm.args.ip.client} ${jvm.args.timeouts}</argLine>
 
                     <systemPropertyVariables>
-                        <jboss.options>-ea ${surefire.system.args}</jboss.options>
+                        <jboss.options>${surefire.system.args}</jboss.options>
```

Comment:
testsuite/pom.xml overrides the surefire.system.args setting from pom.xml and drops the -ea, so without this the PC and HC processes will not have it.